core/ledger/kvledger/txmgmt/statedb/statecouchdb/couchdbutil_test.go

```diff
@@ -68,17 +68,40 @@ func TestCreateCouchDBSystemDBs(t *testing.T) {
 	err = createSystemDatabasesIfNotExist(couchInstance)
 	assert.NoError(t, err, "Error when trying to create system databases")
 
-	db := couchDatabase{couchInstance: couchInstance, dbName: "_users"}
+	db := couchDatabase{couchInstance: couchInstance, dbName: "_replicator"}
 
 	//Retrieve the info for the new database and make sure the name matches
 	dbResp, _, errdb := db.getDatabaseInfo()
-	assert.NoError(t, errdb, "Error when trying to retrieve _users database information")
-	assert.Equal(t, "_users", dbResp.DbName)
+	assert.NoError(t, errdb, "Error when trying to retrieve _replicator database information")
+	assert.Equal(t, "_replicator", dbResp.DbName)
 
-	db = couchDatabase{couchInstance: couchInstance, dbName: "_replicator"}
+	db = couchDatabase{couchInstance: couchInstance, dbName: "_global_changes"}
```

Comment:
Not related to changes made in this PR. We need to understand the last bullet in https://docs.couchdb.org/en/latest/whatsnew/3.0.html#performance better and first bullet in https://docs.couchdb.org/en/latest/whatsnew/3.0.html#deprecated-feature-removal.